pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -1701,6 +1701,41 @@ func TestWaitOnPermit(t *testing.T) {
 	}
 }
 
+func TestListPlugins(t *testing.T) {
+	tests := []struct {
+		name    string
+		plugins *config.Plugins
+		// pluginsCount include queue sort plugin and bind plugin.
+		pluginsCount int
```

Comment:
`pluginSetCount` sounds more appropriate. Otherwise, the 2nd test case would expect 4 "plugins" in total, while in terms of pluginSet count, it's 3.

Reply:
I will modify it later.